ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
+
+
+def meters_to_long(x):
+    # Converts longitude point from Spherical Mercator EPSG:900913 to lat/lon in WGS84 Datum
+    origin_shift = 2 * math.pi * 6378137 / 2.0
+    long = (x / origin_shift) * 180.0
+    return long
+
+
+def meters_to_lat(y):
+    origin_shift = 2 * math.pi * 6378137 / 2.0
+    lat = (y / origin_shift) * 180.0
+    lat = 180 / math.pi * (2 * math.atan(math.exp(lat * math.pi / 180.0)) - math.pi / 2.0)
+    return lat
+
+
+def get_localities(url, username, password):
+    # Access to ArcGIS Online account
+    gis = GIS(url, username, password)
+
+    # Retrieve Field Localities by name, and access first layer in collection
+    localities = gis.content.search(query="title:Field Localities", item_type="Feature Layer")[0].layers[0]
+
+    # Create and return spatially enabled dataframe
+    sdf = pd.DataFrame.spatial.from_layer(localities)
```

Comment:
What is an `sdf`. Better names pls.

Reply:
sdf if the naming convention you find for spatially enabled dataframes. Their based on Pandas dataframes where the convention is "df". 

That being said, I can change the name to be more descriptive!

Reply:
Maybe in that case the naming convention would be whatever the hell you are putting in the `sdf`.